src/lib/components/tables/data-table/terra-data-table.component.scss

```diff
@@ -13,6 +13,20 @@
     --no-result-margin: 0; //in case of a no-result page placed in a table to avoid doubled margins
 
     table {
+        ::ng-deep {
```

Comment:
es wäre denke ich besser das über `:host-context()` in der **_checkbox.scss** zu erreichen